playbooks/roles/tarsnap/README.md

```diff
@@ -17,17 +17,17 @@ Available variables are listed below, along with default values (see `defaults/m
 
     tarsnap_version: 1.0.35
     tarsnap_keyfile: "/root/tarsnap.key"
-    tarsnap_cache: "/usr/local/tarsnap-cache"
+    tarsnap_cache_dir: "/usr/local/tarsnap-cache"
 
-The role-installed `.tarsnaprc` file will automatically exclude the `tarsnap_cache` folder and reference the `tarsnap_keyfile` for backups.  Your play or an out-of-band process is responsible for creating the keyfile and placing it at this location.  See the [standalone](#standalone) section.
+The role-installed `.tarsnaprc` file will automatically exclude the `tarsnap_cache_dir` folder and reference the `tarsnap_keyfile` for backups.  Your play or an out-of-band process is responsible for creating the keyfile and placing it at this location.
```

Comment:
@itsjeyd : I agree. I have taken the liberty here to refactor README, which includes removing links which are not referred anywhere plus a bit of sentence restructuring. I am sure it will need more changes once the implementation part is over.